### PR TITLE
Users.yaml updates. Generalize mirror/upload scripts for multiple ontologies

### PR DESCRIPTION
**Pull request by DoctorBud**
- Improves the mirroring scripts to more easily use different ontology roots
- Updates users.yaml to add Jim and Erin
- Updates Peter Robinson in users.yaml
- adds Matthew Brush to users.yaml
- updates the mirroring scripts that  build the cached OWL files
- adds GO option to mirrorOntologyLocally.sh
- Adds groups.yaml
- Improve uploadMonarchToOntologyCore.sh to use the correct cache and
  catalog, based upon the parameter: go/monarch/hpo

### CODE REVIEW

File: webphenote-config/groups.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+####
+#### Metadata on Monarch groups, derived from GOC groups
+#### See https://github.com/geneontology/go-site/tree/master/metadata
+####
+-
+  label: 'Monarch'
+  id: https://monarchinitiative.org
+-
+  label: 'LBL'
+  id: http://lbl.gov
```
Comment (jnguyenx) [inline webphenote-config/groups.yaml:10]: I think that it has www

**Comment by jnguyenx** [comment]
Looks good to me!

File: webphenote-config/groups.yaml (hunk 1 above)
Comment (DoctorBud) [reply webphenote-config/groups.yaml:10]: I copied my groups.yaml from the [go-site](https://github.com/geneontology/go-site/blob/master/metadata/groups.yaml) version. So maybe should be corrected in both?

**Comment by DoctorBud** [comment]
Going to ignore the groups.yaml issue, since I'm not currently using the Groups feature AFAIK.

**Pull request merged**